index.html

```diff
@@ -17,41 +25,55 @@
       src="https://kit.fontawesome.com/bd800fb8b4.js"
       crossorigin="anonymous"
     ></script>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-217422335-1"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
-    
-    gtag('config', 'UA-217422335-1');
-  </script>
-  <title>Luis Pomare web portfolio</title>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=UA-217422335-1"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "UA-217422335-1");
+    </script>
+    <title>Luis Pomare web portfolio</title>
   </head>
   <body>
     <section class="toolbar">
-      <span class="welcome" >Welcome</span>
-      <nav class="menu-container">
-        <img class="menu" src="images/menu.svg" alt="Navigation menu" />
+      <span class="welcome">Welcome</span>
+      <nav>
+        <img
+          class="menu"
+          src="images/menu.svg"
+          alt="Navigation menu"
+          id="menu-icon"
+        />
+        <ul id="icon-container" class="menu-invisible">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about-tittle">About</a></li>
+          <li><a href="#contact-me">Contact me</a></li>
+        </ul>
```

Comment:
- [ ] Hi, The functionality of the dropdown menu is working perfectly. Although as part of the requirements you need to also ensure that the design matches the reference. Please make sure your design matches the Figma design.

Reference Design | Your Design
-|-
![image](https://user-images.githubusercontent.com/18730106/151050269-00ad2ce7-b994-44ef-8c22-2131025e7e8f.png)|![image](https://user-images.githubusercontent.com/18730106/151050309-e5efcb70-02c4-4ea9-b663-1dd3478554a9.png)